.circleci/config.yml

```diff
@@ -1,15 +1,18 @@
 ---
-version: 2
+version: 2.1
+
+parameters:
+  python-version:
+    type: string
+    default: "3.9"
+  python-module:
+    type: string
+    default: "udata_hydra"
```

Comment:
Yes! The rationale is also that otherwise the module name would be buried in a single command and we copy paste the conf to another repo it would be easier to spot that way. Plus we could easily imagine using this var elsewhere.